win32/Lib/win32serviceutil.py

```diff
@@ -50,8 +50,14 @@ def LocatePythonServiceExe(exe=None):
     # pywin32 installed it next to win32service.pyd (but we can't run it from there)
     maybe = os.path.join(os.path.dirname(win32service.__file__), exe)
     if os.path.exists(maybe):
-        print(f"copying host exe '{maybe}' -> '{correct}'")
-        win32api.CopyFile(maybe, correct)
+        print(f"moving host exe '{maybe}' -> '{correct}'")
+        # Handle case where MoveFile() fails. Particularly if destination file
+        # has a resource lock and can't be replaced by src file
+        try:
+            win32api.MoveFile(maybe, correct)
```

Comment:
I believe this will fail if the target exists - but we do want to update the target in that case. So I think you maybe want `MoveFileEx` with `MOVEFILE_REPLACE_EXISTING`?

Reply:
Ah yeah, that does look better.
I do find in most cases this move would fail on my deployments as the destination file is in use by several perpetually running services.  I guess the correct approach is to stop these services to free up the destination resource.

Reply:
I've applied the recommended changes Mark :)